yew/src/virtual_dom/vtag.rs

```diff
@@ -133,7 +134,7 @@ impl VTag {
         }
     }
 
-    /// Returns tag of an `Element`. In HTML tags are always uppercase.
+    /// Returns tag of an `Element` in lowercase
```

Comment:
I think it's only guaranteed to be lowercase when using the macro right?

Reply:
Right, I'll revert this.